package.json

```diff
@@ -7,24 +7,17 @@
     "build": "yarn bazel build ...",
     "deploy:builds": "ts-node ./build/deploy-build.ts",
     "deploy:preview": "ts-node ./build/deploy-preview.ts",
-    "test:unit": "node ./tests.js",
```

Comment:
What about keeping these tests scripts?
I think having to run the tests via BAZEL locally would be an extra hurdle to take for (new) contributors.

Reply:
Good point. Bazel doesn't play very nice with Windows users also. We'll keep them in the short term as a fallback.